smac/initial_design/initial_design.py

```diff
@@ -82,6 +83,8 @@ def __init__(self,
             specify if target algorithm has to be run once on the first configuration before the intensify call
         fill_random_configs: bool
             fill budget with random configurations if initial incumbent sampling returns only 1 configuration
+        init_budget : int, optional
+            Maximal initial budget (alternative to ``n_configs_x_params``)
```

Comment:
It is unclear from this documentation what happens if both or None are given

Reply:
It is also unclear what happens if, e.g. n_configs_x_params is higher/lower than the initial budget

Reply:
I just extended the docstrings to clarify the behavior.